### PR TITLE
Allow AWS regions as well as zones

### PR DESCRIPTION
**Pull request by ehankland**
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/issues/655

### CODE REVIEW

**Comment by ehankland** [comment]
Also addresses https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/issues/493

**Comment by cmccoy** [comment]
Could you add a few tests?
Should we consider a regex validator?

**Comment by ehankland** [comment]
Done - PTAL

File: perfkitbenchmarker/providers/aws/util.py [hunk 1]
```diff
@@ -23,6 +26,21 @@
 FLAGS = flags.FLAGS
 
 
+def IsRegion(zone_or_region):
+  """Returns whether "zone_or_region" is a region."""
+  if not re.match('[a-z]{2}-[a-z]+-[0-9][a-z]?\Z', zone_or_region):
```
Comment (cmccoy) [inline perfkitbenchmarker/providers/aws/util.py]: Make this a raw string.
Nit: I think `$` is more commonly used in the codebase to identify the end of a string.

**Comment by cmccoy** [comment]
One comment, otherwise LGTM.

File: perfkitbenchmarker/providers/aws/util.py (hunk 1 above)
Comment (ehankland) [inline perfkitbenchmarker/providers/aws/util.py]: Done

**Pull request merged**